<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>spot_diagnostics_dialog</class>
 <widget class="QDialog" name="spot_diagnostics_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Spot Diagnostics</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="quantity_label">
     <property name="text">
      <string>Quantity:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="quantity">
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="bounds_label">
     <property name="text">
      <string>Bounds:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QDoubleSpinBox" name="bounds">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>0.000100000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="histogram_bins_label">
     <property name="text">
      <string>Histogram bins:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QSpinBox" name="histogram_bins">
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="detector_label">
     <property name="text">
      <string>Detector:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QComboBox" name="detector">
     <property name="styleSheet">
      <string notr="true">combobox-popup: 0;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QCheckBox" name="show_all_grains">
     <property name="text">
      <string>Show all grains</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QLabel" name="grain_id_label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Grain ID:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QComboBox" name="grain_id">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">combobox-popup: 0;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="11">
    <spacer name="horizontal_spacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="12">
    <layout class="QVBoxLayout" name="canvas_layout"/>
   </item>
   <item row="2" column="0" colspan="12">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>quantity</tabstop>
  <tabstop>bounds</tabstop>
  <tabstop>histogram_bins</tabstop>
  <tabstop>detector</tabstop>
  <tabstop>show_all_grains</tabstop>
  <tabstop>grain_id</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>spot_diagnostics_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>700</x>
     <y>874</y>
    </hint>
    <hint type="destinationlabel">
     <x>700</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
